### ISSUE TITLE
Use Literate.jl for up keeping examples

### ISSUE THREAD

**Issue by mirestrepo** [JuliaHealth/BioMedQuery.jl#17]
[Literate.jl](https://github.com/fredrikekre/Literate.jl) - neat way to marry notebooks + documenter

**Comment by mcmcgrath13** [comment]
Updated docs/examples to use Literate.  It's great!  Once BioServices fix is in place, we can merge into master.

**Comment by mirestrepo** [comment]
Merged #22. However waiting on a successful travis build to close

**Comment by mirestrepo** [comment]
Missing:

* Call literate from within docs/make.jl

**Comment by mcmcgrath13** [comment]
[PR #28](https://github.com/bcbi/BioMedQuery.jl/pull/28)

**Comment by DilumAluthge** [comment]
Since PR #28 has been merged, can we close this issue?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by DilumAluthge** [comment]
@mcmcgrath13 @mirestrepo 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by mirestrepo** [comment]
yeap
